EMS/common-bundle/src/Common/CoreApi/Endpoint/Search/Search.php

```diff
@@ -118,4 +118,40 @@ public function getDocument(string $index, ?string $contentType, string $id, arr
             'sources-excludes' => $sourcesExcludes,
         ])->getData());
     }
+
+    /**
+     * @param string[] $aliases
+     *
+     * @return array<string, array<int, string>>
+     */
+    public function getIndicesForContentTypes(array $aliases): array
+    {
+        $indices = $this->client->post('/api/search/indices-for-content-type', [
+            'aliases' => $aliases,
+        ])->getData()['indices'] ?? null;
+        if (!\is_array($indices)) {
+            throw new \RuntimeException('Unexpected: search must be an array');
+        }
+
+        return $indices;
+    }
+
+    /**
+     * @param string[] $words
+     *
+     * @return string[]
+     */
+    public function filterStopWords(string $index, string $analyzer, array $words): array
```

Comment:
Method should be added to: EMS\CommonBundle\Contracts\CoreApi\Endpoint\Search\SearchInterface

Reply:
713326cd1c29d9d19f63abc95baba545975667b2